### ISSUE TITLE
Flaky test: RazorPagesTemplate_IndividualAuth_Works_NetCore

### ISSUE THREAD

**Issue by ryanbrandenburg** [aspnet/Templating#639]
This test [fails](http://aspnetci/viewLog.html?buildId=515544&buildTypeId=Releases_22xPublic_Win2012) occasionally with the following error:
```
System.Exception : Process exited with code 1
StdErr: 
StdOut: Build failed.
   at Templates.Test.Helpers.ProcessEx.WaitForExit(Boolean assertSuccess) in /_/test/Templates.Test/Helpers/ProcessEx.cs:line 150
   at Templates.Test.TemplateTestBase.RunDotNetEfCreateMigration(String migrationName) in /_/test/Templates.Test/Helpers/TemplateTestBase.cs:line 130
   at Templates.Test.RazorPagesTemplateTest.RazorPagesTemplate_IndividualAuthImpl(String targetFrameworkOverride, Boolean useLocalDB, Boolean noHttps) in /_/test/Templates.Test/RazorPagesTemplateTest.cs:line 104
------- Stdout: -------
==> C:\b\w\33bdfc1cae7b2a38\.dotnet\x64\dotnet.exe new razor --debug:custom-hive "C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\.templateengine" --auth Individual [C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede]
The template "ASP.NET Core Web App" was created successfully.
This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore-template-3pn-210 for details.
Processing post-creation actions...
Running 'dotnet restore' on C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj...
  Restoring packages for C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj...
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102: Unable to find package Microsoft.AspNetCore.App with version (>= 2.2.0)
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 48 version(s) in https://dotnet.myget.org/F/dotnet-core/api/v3/index.json [ Nearest version: 2.1.3-rtm-30927 ]
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 46 version(s) in https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json [ Nearest version: 2.1.3-rtm-30927 ]
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 6 version(s) in nuget.org [ Nearest version: 2.1.2 ]
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 1 version(s) in C:\b\w\33bdfc1cae7b2a38\.deps\build\ [ Nearest version: 2.2.0-preview1-34790 ]
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 0 version(s) in C:\b\w\33bdfc1cae7b2a38\artifacts\build\
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 0 version(s) in https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 0 version(s) in https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 0 version(s) in https://dotnet.myget.org/F/roslyn/api/v3/index.json
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 0 version(s) in https://vside.myget.org/F/vsmac/api/v3/index.json
C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj : error NU1102:   - Found 0 version(s) in https://vside.myget.org/F/vssdk/api/v3/index.json
  Generating MSBuild file C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\obj\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj.nuget.g.props.
  Generating MSBuild file C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\obj\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj.nuget.g.targets.
  Restore failed in 942.9 ms for C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp2.2\TestTemplates\AspNet.Template.fc9ebed156044abf876070b27962eede\AspNet.Template.fc9ebed156044abf876070b27962eede.csproj.
Restore failed.
Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet resto
...
```

CC @Eilon,@muratg,@mkArtakMSFT

This issue was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
Please use this workflow to address this flaky test issue, including checking applicable checkboxes and filling in the applicable "TODO" entries:

* Is this test failure caused by product code flakiness? (Either this product, or another product this test depends on.)
  * [ ] File a bug against the product (TODO: Link to other bug)
  * Is it possible to change the test to avoid the flakiness?
    * Yes? Go to the "Change the test!" section.
    * No?
      * [ ] Disable the test (TODO: Link to PR/commit)
      * [ ] Wait for other bug to be resolved
      * [ ] Wait for us to get build that has the fix
      * [ ] Re-enable our test (TODO: Link to PR/commit)
      * [ ] Close this bug

* Is it that the test itself is flaky? This includes external transient problems (e.g. remote server problems, file system race condition, etc.)
  * Is there is a way to change our test to avoid this flakiness?
    * Yes? Change the test!
      * [ ] Change the test to avoid the flakiness, for example by using a different test strategy, or by adding retries w/ timeouts (TODO: Link to PR/commit)
      * [ ] Run the test 100 times locally as a sanity check.
      * [ ] Close this bug
    * No?
      * [ ] Delete the test because flaky tests are not useful (TODO: Link to PR/commit)

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515544&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515782&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515977&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515925&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by Eilon** [comment]
@ryanbrandenburg all yours.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516180&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516660&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516645&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516549&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516714&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516700&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516834&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516957&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516943&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517017&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517196&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517757&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517599&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518163&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518436&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518450&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518510&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518585&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518699&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518926&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519073&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519091&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519450&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519339&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519558&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519492&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519586&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519572&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519624&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519689&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=520895&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=522188&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
MvcTemplate_IndividualAuth_UsingLocalDB_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=523211&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=524711&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_IndividualAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=524676&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
Judging by the frequency with which this failed before I think this was fixed by my recent blast of fixes to Templating. I'm not exactly sure what I did to fix it though. @ajaybhargavb I'll leave it to you to decide if you want to dig more or call it fixed.

**Comment by ajaybhargavb** [comment]
Fixed by @ryanbrandenburg with d95f9716932cae55933c0a4700fec3513c55c92f
